packages/core/cms/faststore/sections.json

```
@@ -1607,6 +1607,59 @@
               "default": true
             }
           }
+        },
+        "emptyGallery": {
+          "title": "Empty Gallery",
+          "type": "object",
+          "properties": {
+            "title": {
+              "title": "Icon",
+              "type": "string",
+              "default": "Nothing matches with your search"
+            },
+            "topButton": {
+              "title": "Top Button",
+              "type": "object",
+              "properties": {
+                "text": {
+                  "type": "string",
+                  "title": "Top Button Text",
+                  "default": "Browse Offers"
+                },
+                "href": {
+                  "type": "string",
+                  "title": "URL",
+                  "default": "/office"
+                },
+                "iconName": {
+                  "title": "Icon Name",
+                  "type": "string",
+                  "default": "CircleWavyWarning"
+                }
```

```suggestion
                "icon": {
                  "title": "Icon",
                  "type": "string",
                  "enumNames": ["Circle Wavy Warning"],
                  "enum": ["CircleWavyWarning"]
                }
```

We just have a set of icons available for now 🙇 So let's keep like that for now, ok?